packages/web/static/locales/en/faucet.json

```diff
@@ -34,7 +34,7 @@
   "testnetForum": "Alfajores Testnet Forum",
   "testnetAddress": "Testnet Address",
   "getStarted": "Get Started",
-  "done": "Done",
+  "done": "Done. SMS Message \n arriving shortly",
```

Comment:
Can we say `SMS message will arrive in 1-2 minutes...` to give more realistic expectations?

Reply:
let me see what the content team thinks

Reply:
updated to SMS arriving in 1 - 2 minutes